source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
Why this doesn't work? If the developer uses Likely as a Commonjs module
than it's the first branch of "if", and we create here Likely instance
without any browser specific behavior. In the other branch we sure that the
bundle wasn't used within Commonjs infrastructure, so we include here ant
additional functionality like auto-init, cause we're sure that the bundle
is used by including it as a separate script in browser. Seems it doesn't
break any compatibility. The only problem I see here is that bundle build
should be configured somehow differs from default. Where I'm wrong?

воскресенье, 22 мая 2016 г. пользователь Andrew Romanov написал:

> In source/likely.js
> <https://github.com/ilyabirman/Likely/pull/58#discussion_r64153025>:
>
> > @@ -2,5 +2,6 @@
> >
> >  var likely = require('./index.js');
> >
> > -window.likely = likely;
> >  window.addEventListener('load', likely.initiate);
>
> Ok 👍
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ilyabirman/Likely/pull/58/files/2d7471a850b99ed55851d4202bf0f087aa6c6b65#r64153025>
>


-- 
Виктор.